### ISSUE TITLE
31_fossil/timeDepGrades and 31_fossil/MOFEX dysfunctional 

### ISSUE THREAD

**Issue by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [remindmodel/remind#1459]
As of pik-piam/mrremind#107 (September 9 2021), required input files `f31_decoffset.cs4r`, `f31_Xport.cs4r`, `f31_extraseed.cs4r` are not generated anymore.
Maybe it's time to either fix this, or remove the realisations.
See pik-piam/mrremind#447.

**Comment by nicobauer** [comment]
Nope. Please no removal!

**Comment by LaviniaBaumstark** [comment]
I agree that those realizations either need to be reactivated or deleted.

**Comment by nicobauer** [comment]
It will be reactivated.
